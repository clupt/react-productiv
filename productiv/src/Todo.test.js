import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

const testTodo = {
  id: 1,
  title: "test-todo",
  description: "test-description",
  priority: 1,
};

describe("todo component", function () {
  test("renders without crashing", function () {
    render(<Todo todo={testTodo} />);
  });

  test("renders a todo", function () {
    const { container, debug } = render(<Todo todo={testTodo} />);

    expect(container.querySelector(".Todo")).toBeInTheDocument();
  });
});

it("matches snapshot", function () {
  const { container } = render(<Todo todo={testTodo} />);

  expect(container).toMatchSnapshot();
});
