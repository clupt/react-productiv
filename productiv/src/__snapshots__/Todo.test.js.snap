// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        test-todo
      </b>
       
      <small
        id="priority"
      >
        1
      </small>
    </div>
    <div>
      <small>
        test-description
      </small>
    </div>
  </div>
</div>
`;
